config/config.go

```diff
@@ -246,3 +246,12 @@ func GetEtcdServerAddrs() []string {
 	}
 	return []string{"http://localhost:2379"}
 }
+
+func IsTLSenabled() bool {
```

Comment:
This could be moved to common/tls.go too

I would prefer to use a boolean in the configuration to enable/disable TLS, something like "ws_use_tls: true"
That makes debugging tls easier : SKYDIVE_WS_USE_TLS=false /usr/bin/skydive allinone ...